src/structs.jl

```diff
@@ -537,7 +546,8 @@ mutable struct MPSGEModel <:AbstractMPSGEModel
     commodities::Dict{ScalarCommodity,Vector{ScalarSector}} #Generated on model build
     auxiliaries::Dict{ScalarAuxiliary, AuxConstraint}
     silent::Bool
-    MPSGEModel() = new(Dict(),Model(PATHSolver.Optimizer; add_bridges = false),Dict(),Dict(),Dict(),Dict(),false)
+    numeraire::Union{MPSGEVariable,Missing}
+    MPSGEModel() = new(Dict(),direct_model(PATHSolver.Optimizer()),Dict(),Dict(),Dict(),Dict(),false,missing)
```

Comment:
This a small but _very_ important change. We need to use `direct_model` rather than `Model`. This allows us to query variables when the model is "dirty". 

Using `Model` if we fix a variable we can no longer call `value`, it raises a "Model modified since last call to `optimize!`" error. `direct_model` solves this.